packages/blade/src/components/Carousel/Carousel.native.tsx

```diff
@@ -154,7 +155,7 @@ const Carousel = ({
     },
   );
 
-  React.useEffect(() => {
+  useUpdateEffect(() => {
```

Comment:
Rather call it - `useFirstRenderEffect()`?

Reply:
No it's not first render effect. It skips first render and only runs un subsequent renders thus called "update effect". 

"useUpdateEffect" is a very commonly used name for this hook throughout different libraries.

Reply:
I thought you're checking first mount 🤦